src/Sniffer.php

```diff
@@ -39,9 +39,15 @@ public function preCommit()
 
         echo 'Running PHP CodeSniffer in ' . $this->root . PHP_EOL;
         $sniffer = new PHP_CodeSniffer_CLI();
+        @$config = $this->getExtraKey('php-composter-phpcs-wpcs', [
+            'standard' => 'WordPress-Extra'
+        ]);
 
         ob_start();
-        $numErrors = $sniffer->process(array('standard' => 'WordPress-Extra', 'files' => $files));
+        $numErrors = $sniffer->process([
```

Comment:
Did you think about merging the `$files` into the `$config` array here? This way, one could even pass other PHPCS/WPCS settings through the configuration in `composer.json`. Not sure there's much that makes sense, though...